manifests/e/Ergosoft/ErgoFAKT_V5/4.9.2.5294/Ergosoft.ErgoFAKT_V5.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Ergosoft.ErgoFAKT_V5
+PackageVersion: 4.9.2.5294

```

Comment:
```suggestion
PackageVersion: 4.9.025294
```

Reply:
4.9.2.5294 does not reflect the msi-installer version, but ist the correct version of the product, so this is intended.

Reply:
Unfortunately, the version number must match Control Panel.

If you think this is incorrect then please update your software with the correct version and update `InstallerUrl` and `InstallerSha256`.